builder/dockerfile/internals.go

```diff
@@ -504,6 +504,23 @@ func (b *Builder) create() (*daemon.Container, error) {
 	}
 	b.runConfig.Image = b.image
 
+	var mounts []string
+	potentialMounts := []string{
+		"/usr/bin/qemu-arm-static",
+		"/usr/bin/qemu-aarch64-static",
+	}
+
+	for _, mount := range potentialMounts {
+		if _, err := os.Stat(mount); err == nil {
+			toAdd := fmt.Sprintf("%s:%s:ro", mount, mount)
+			mounts = append(mounts, toAdd)
+			fmt.Fprintf(b.Stdout, " ---> Adding bind mount during build: %s\n", toAdd)
```

Comment:
These will be printed to the terminal on a build; not sure if a) we want to clutter the output and b) want to advertise that we're doing this.